src/libraries/Microsoft.PowerFx.Core/Public/Values/BlobValue.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics.Contracts;
+using System.Text;
+using Microsoft.PowerFx.Core.IR;
+
+namespace Microsoft.PowerFx.Types
+{
+    public class BlobValue : PrimitiveValue<string>
```

Comment:
>PrimitiveValue<string> [](http://example.com/codeflow?start=29&length=22)

Changing base class later is a big deal, so we should be confident this is the pattern we want for future scenarios.